src/FormBuilder/resources/js/form-builder/src/settings/form-summary/page-slug.js

```diff
@@ -11,71 +11,69 @@ const {
-                      label={'URL'}
-                      value={'/donations/' + editedSlug}
-                      onChange={() => null}
-                      onClick={onToggle}
-                      aria-expanded={isOpen}
-                  />
-              )}
-              renderContent={({onClose}) => (
-                  <div style={{minWidth: 'calc(var(--givewp-sidebar-width) - 48px)'}}>
-                      <div style={{display: 'flex', justifyContent: 'space-between', alignItems: 'center'}}>
-                          <strong style={{fontSize: '14px'}}>{'URL'}</strong>
-                          <Button onClick={onClose}>
-                              <Icon icon={close} size={14}></Icon>
-                          </Button>
-                      </div>
-                      <TextControl
-                          label={'Permalink'}
-                          value={editedSlug}
-                          autoComplete="off"
-                          spellCheck="false"
-                          onChange={(newPageSlug) => {
-                              setEditedSlug(newPageSlug);
-                          }}
-                          help={'The last part of the URL.'}
-                          onBlur={() => updateSlug()}
-                      />
-                      <div>
-                          <strong style={{fontSize: '14px'}}>View Page</strong>
-                      </div>
-                      <ExternalLink href={permalink} style={{fontSize: '14px'}}>
-                          {sprintf('%s/%s', rewriteSlug, editedSlug)}
-                      </ExternalLink>
-                  </div>
-              )}
-          />
-      )
-  );
+    return (
+        !!isEnabled && (
+            <Dropdown
+                className="my-container-class-name"
+                contentClassName="givewp-sidebar-dropdown-content"
+                popoverProps={{placement: 'bottom-start'}}
+                focusOnMount={'container'}
+                renderToggle={({isOpen, onToggle}) => (
+                    <TextControl
+                        label={'URL'}
+                        value={'/donations/' + editedSlug}
+                        onChange={() => null}
+                        onClick={onToggle}
+                        aria-expanded={isOpen}
+                    />
+                )}
+                renderContent={({onClose}) => (
+                    <div style={{minWidth: 'calc(var(--givewp-sidebar-width) - 48px)'}}>
+                        <div style={{display: 'flex', justifyContent: 'space-between', alignItems: 'center'}}>
+                            <strong style={{fontSize: '14px'}}>{'URL'}</strong>
+                            <Button onClick={onClose}>
+                                <Icon icon={close} size={14}></Icon>
+                            </Button>
+                        </div>
+                        <TextControl
+                            label={'Permalink'}
```

Comment:
@JoshuaHungDinh I noticed that a few strings in this component are not translated. Is it intentional?

Reply:
I don't believe so. Resolved: 12f766a92ba4fbacd9645c3ce63d7946a722bfb1